arcadedefs/idolmtower.arcadedef

```diff
@@ -0,0 +1,17 @@
+{
+	"id": "idolm",
```

Comment:
This should have its very own id: `idolmtower` would make sense since you've used that name for this file.

Reply:
I was under the impression that the game ID doesn't change since it's the same HDD and game code, it just starts in one state or another. I'll change it if it wasn't the case.